tests/strings/strings.js

```diff
@@ -33,3 +33,7 @@ foo(`a long string ${ 1 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + 3 + 2 + ( function
 
   return x;
  })() + 3 + 2 + 3 + 2 + 3 } with expr`);
+
+pipe.write(
```

Comment:
btw, strings/strings.js was supposed to be around escaping stuff, not those template literal edge cases. Could you move those in another file/folder?